### ISSUE TITLE
Long y-axis labels on line chart are clipped

### ISSUE THREAD

**Issue by johnmccutchan** [google/charted#142]
0.0
200000.0
400000.0
600000.0
800000.0
1000000.0 <-- clipped to '000000.0'

**Comment by psunkari** [comment]
For performance reasons, we require that the right font (that is used in CSS) be configured in ChartAxisTheme.  Currently the only way to do it is by extending from the default theme and customizing it.

Please ping me if you need more help here.

**Comment by psunkari** [comment]
Also, we have a plan to implement DOM based text metrics, but that may take a couple of weeks.

**Comment by johnmccutchan** [comment]
Any update on an ETA for DOM based text metrics?

**Comment by johnmccutchan** [comment]
@psunkari ping.

**Comment by psunkari** [comment]
DOM based text metrics will take time.
However, did you attempt to customize the theme?

**Comment by johnmccutchan** [comment]
@psunkari No, because you said in September that DOM based text was only a couple of weeks away. Can I get an updated ETA?
